src/utils/database/users_table.py

```diff
@@ -0,0 +1,74 @@
+"""
+User Table module
+"""
+
+from typing import List
+from tinydb import TinyDB, where
+from tinydb.table import Document
+from .db_decorators import validate_insert
+
+class UsersTable():
+    """
+    Table to hold all registered users
+    """
+    TABLE_LEETCODE_USER_FIELDS = ["discord_id", "leetcode_id"]
+    TABLE_LEETCODE_USER = "Leetcode_User"
```

Comment:
move to __init__() as self.table_fields and self.table_name

Reply:
Renamed but kept outside of constructor due to issues with the validate_insert decorator.